server/src/test/java/io/spine/server/model/MethodExceptionCheckerShould.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2018, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.model;
+
+import io.spine.test.Tests;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+
+import static io.spine.server.model.MethodExceptionChecker.forMethod;
+
+/**
+ * @author Dmytro Kuzmin
+ */
+public class MethodExceptionCheckerShould {
+
+    @Test(expected = NullPointerException.class)
+    public void do_not_accept_null_method() {
+        //noinspection ResultOfMethodCallIgnored
+        forMethod(Tests.<Method>nullRef());
```

Comment:
Please use `NullPointerTester` here and in similar cases.

Reply:
Also, we do not usually use the IDEA's suppression-by-comment feature, since it's not compatible with any standalone style checker. Please remove the usages in this files and scan for them all over the changeset. If necessary, use `@SuppressWarnings("...")` instead.